### PR TITLE
docs(Label): update component description

### PR DESCRIPTION
**Pull request by goshander**

### CODE REVIEW

File: src/components/Label/__stories__/Label.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Label component is best designed to display one line text information with different color indication of its importance.
+
+## Appearance
+
+Label can be displayed with multiple styled combination
+
+-   theme (`normal`, `info`, `danger`, `warning`, `success`, `unknown`, `clear`)
+-   type (`default`, `copy`, `close`)
+-   style (`default`, `rounded`)
```
Comment (amje) [inline src/components/Label/__stories__/Label.mdx:53]: This is deprecated, should be removed

File: src/components/Label/__stories__/examples/LabelExampleView/LabelExampleView.tsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                    Disabled
+                </Label>
+                <Label size="s" interactive>
+                    Interactive
+                </Label>
+            </DocsExample>
+        </Unstyled>
+    );
+}
+LabelExampleSize.code = `
```
Comment (amje) [inline src/components/Label/__stories__/examples/LabelExampleView/LabelExampleView.tsx:164]: Shouldn't it be `LabelExampleState.code`?

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Label/__stories__/Label.mdx (hunk 1 above)
Comment (goshander) [reply src/components/Label/__stories__/Label.mdx]: removed

File: src/components/Label/__stories__/examples/LabelExampleView/LabelExampleView.tsx (hunk 2 above)
Comment (goshander) [reply src/components/Label/__stories__/examples/LabelExampleView/LabelExampleView.tsx]: Oh, thank you, fixed

**Review by amje — APPROVED** [review]

**Pull request merged**